bin/qless-py-worker

```diff
@@ -1,10 +1,11 @@
 #! /usr/bin/env python
 
-import psutil
+import multiprocessing as mp
 import argparse
 
 # First off, read the arguments
-parser = argparse.ArgumentParser(description='Run qless workers.')
+parser = argparse.ArgumentParser(description='Run qless workers.',
```

Comment:
This change also appears in a different pull request. Did it creep in here unintentionally?